python/ray/tests/test_placement_group_5.py

```diff
@@ -0,0 +1,96 @@
+import pytest
```

Comment:
Can you also update https://github.com/ray-project/ray/blob/2302b4eea87c480699797e4bf770d46d0b758379/python/ray/tests/BUILD#L96?